content/docs/glossary/claimable-balance.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Claimable Balance
+order:
+---
+
+import { CodeExample } from "components/CodeExample";
+
+Claimable Balances can be used to "split up" a payment into two parts, which allows the sending to only depend on the sending account, and the receipt to only depend on the receiving account. An account can initiate the "send" by creating a ClaimableBalanceEntry with [Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance), and then that entry can be claimed by the claimants specified on the ClaimableBalanceEntry at a later time with [Claim Claimable Balance](../start/list-of-operations.mdx#claim-claimable-balance). 
+
+## Relevant operations
+
+[Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance)
+
+### Parameters
+
+1. Asset - Asset that will be held in the ClaimableBalanceEntry.
+
+2. Amount - Amount of asset stored in the ClaimableBalanceEntry.
+
+3. List of Claimants - A Claimant is an object that holds both the account that can claim the ClaimableBalanceEntry, and a ClaimPredicate that must evaluate to true for the claim to succeed. A ClaimPredicate is a recursive data structure that can be used to construct complex conditionals. Here are some examples -
+   - Can claim at anytime - `CLAIM_PREDICATE_UNCONDITIONAL`
+   - Can claim if the ledger close time in which the claim was included is before X seconds + the ledger close time in which the ClaimableBalanceEntry was created  -  `CLAIM_PREDICATE_BEFORE_RELATIVE_TIME(X)`
+   - Can claim if the ledger close time in which the claim was included is before X (Unix timestamp) - `CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)`
+   - Can claim if the ledger close time in which the claim was included is after X seconds + the ledger close time in which the ClaimableBalanceEntry was created - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_RELATIVE_TIME(X))`
+   - Can claim if the ledger close time in which the claim was included is at or after X (Unix timestamp) - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X))`
+   - Can claim between X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_AND(CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)), CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
+   - Can claim outside X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_OR(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X), CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
```

Comment:
There may be value in dropping the `CLAIM_PREDICATE_` prefix to improve readability, perhaps with a clarifying message about how SDKs will likely still have a prefix?

Reply:
I think your examples are really awesome! Definitely wish I had these before writing Horizon tests.

One suggestion to lower verbosity a tad:
> if the ledger close time in which the claim was included 

vs.

> if the close time of the ledger including the claim

Thoughts? Also, italics on the differences between each example (like _between_ vs. _outside_ in the last two) would be a helpful visual cue!

Reply:
Removed `CLAIM_PREDICATE_` and followed your other suggestions as well. Definitely easier to read now. Thanks!